assets/app/views/events.html

```diff
@@ -39,9 +39,13 @@
                     <td class="hidden-sm hidden-md" data-title="Kind">
                       {{event.involvedObject.kind}}</td>
                     <td class="hidden-sm hidden-md" data-title="Reason">
+                      <event-icon type="{{event.type}}"></event-icon>
```

Comment:
I originally wanted to do it that way but then I figured out that it would be better to have two templates anyway since if we would have only one there would be a lot of `ng-if` conditions, since there is quite a lot of differences between those two tables